app/client/components/payment/directDebitDisplay.tsx

```diff
@@ -61,93 +59,67 @@ export interface DirectDebitDisplayProps
   inErrorState?: boolean;
 }
 
-export const DirectDebitInlineDisplay = (mandate: DirectDebitDisplayProps) => (
-  <div
-    css={{
-      display: mandate.inline ? "inline-flex" : "flex",
-      alignItems: "top"
-    }}
-  >
-    <i
-      css={css`
-        margin-top: 2px;
+export const DirectDebitInlineDisplay = (props: DirectDebitDisplayProps) => (
```

Comment:
I notice this is now called DirectDebit**Inline**Display despite most of the places being not inline (i.e. spread over multiple lines) - I wonder if it should go back to the old name (with `inline` prop for forcing it inline)

... unless the one inline screenshot is out of date

Reply:
Update payment initial step screenshot updated. The only place the inline version of the direct debit display component is now used is in the flow start multiple product choice display, which we will be refactoring out in another pr shortly

Reply:
since we're going to simplify/delete `flowStartMultipleProductDetailHandler.tsx` very soon you can ignore the above.